src/main/java/models/Constants.java

```diff
@@ -0,0 +1,25 @@
+package models;
```

Comment:
If we don't like this package name, I would love to know other suggestions!

Reply:
I feel like this could be named `utils` instead of `models` but this is my quick suggestion

Reply:
yeah, utils may be okay. at least it's better than models. thanks for the suggestion!